source/_includes/puppet-collections/_puppet_collection_1_yum.md

```diff
@@ -0,0 +1,16 @@
+To enable the Puppet Collection 1 repository, first choose the package based on your operating system and version. The packages are located in the [`yum.puppetlabs.com`](https://yum.puppetlabs.com) repository and named using the following convention:
+
+    puppetlabs-release-<COLLECTION>-<OS ABBREVIATION>-<OS VERSION>.noarch.rpm
+
+For instance, the package for Puppet Collection 1 on Red Hat Enterprise Linux 7 (RHEL 7) is `puppetlabs-release-pc1-el-7.noarch.rpm`.
+
+Next, use the `rpm` tool as root with the `upgrade` (`-U`) flag, and optionally the `verbose` (`-v`), and `hash` (`-h`) flags:
```

Comment:
This line is notable as there were conflicts between the guides and our versioned docs about whether to use the upgrade or install flags.